src/components/InputUserEmailVerify.tsx

```
@@ -209,8 +212,13 @@ const InputUserEmailVerify = forwardRef<HTMLInputElement, InputType>(
 			// Check if user is changing email address
 			if (e.target.value !== user.email) {
 				setVerified(false);
+				props.verifiedSaveButton && props.verifiedSaveButton(false);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)